--- authnzerver/tests/test_auth_creation.py	2022-02-02 14:39:05.977568648 +0000
+++ authnzerver/tests/test_auth_creation_processedpatch_02380cb7154.py	2022-02-03 05:46:12.733689514 +0000
@@ -22,6 +22,15 @@ def get_test_authdb():
 
 
 def test_create_user():
+    user_payload = {'full_name': 'Test User', 'email': 'testuser-passcheck@test.org', 'password': 'aROwQin9L8nNtPTEMLXd', 'pii_salt': 'super-secret-salt', 'reqid': 1}
+    currproc = mp.current_process()
+    if getattr(currproc, 'authdb_engine', None):
+        currproc.authdb_engine.dispose()
+        del currproc.authdb_engine
+    try:
+        os.remove('test-passcheck.authdb.sqlite')
+    except Exception:
+        pass
     '''
     This runs through various iterations of creating a user.
 
